airflow/models/dag.py

```diff
@@ -2002,6 +2007,18 @@ def get_all(cls, session) -> dict[str, dict[str, str]]:
         return dag_links
 
 
+bundle = DagBundle.model_construct(
+    name="my-bundle",
+    classpath="airflow.dag_processing.bundles.git.GitDagBundle",
+    kwargs={
+        "repo_url": "git@github.com:dstandish/my-dags.git",
+        "tracking_ref": "main",
+        "subdir": "dags",
+    },
+    version="dd4399e",
+)
+
```

Comment:
this is just to fake the data since the parsing side of things is not in main yet